nekoyume/Assets/_Scripts/State/LocalLayerModifier.cs

```diff
@@ -140,10 +127,16 @@ public static void AddItem(Address avatarAddress, Guid guid, bool resetState = t
 
         public static void AddItem(
             Address avatarAddress,
-            Dictionary<HashDigest<SHA256>, int> idAndCountDictionary,
-            bool resetState = true)
+            HashDigest<SHA256> fungibleId,
+            int count = 1,
+            bool resetState = default)
```

Comment:
I know the bool default value is false, is this okay?

Reply:
Yes it is. I have fixed all other arguments.